Tasks/AutomatedAnalysisV0/task.json

```diff
@@ -0,0 +1,468 @@
+        },
+        {
+            "name": "classifier",
+            "type": "pickList",
+            "label": "Classifier",
+            "defaultValue": "mannWhitney",
+            "required": true,
+            "options": {
+                "mannWhitney": "mannWhitney"
+            },
+            "groupName": "classificationInputs",
+            "helpMarkDown": ""
+        },
+        {
+            "name": "metricGroups",
+            "type": "multiLine",
+            "label": "Metric groups",
+            "required": false,
+            "properties": {
+                "resizable": "true",
+                "rows": "4"
+            },
+            "groupName": "classificationInputs",
+            "helpMarkDown": "Multiline input where each line contains a number corresponding to the line of same index in the expressionTempaltes input. Group numbering starts from 1,2,3… If this input is left unspecified, all metrics are clubbed in a single group (Group 1)"
+        },
+        {
+            "name": "groupWeights",
+            "type": "multiLine",
+            "label": "Group weights",
+            "required": false,
+            "properties": {
+                "resizable": "true",
+                "rows": "4"
+            },
+            "groupName": "classificationInputs",
+            "helpMarkDown": "Multiline input where each line denotes a weight to be used for each unique group specified(1,2,3..) while computing the final score. 0 < groupWeighti <= 1. Sum of all groupWeights should be equal to 1. If this input is left unspecified, all groups are given equal weights"
+        },
+        {
+            "name": "direction",
+            "type": "multiLine",
+            "label": "Direction",
+            "required": false,
+            "properties": {
+                "resizable": "true",
+                "rows": "4"
+            },
+            "groupName": "classificationInputs",
+            "helpMarkDown": "Multiline input where each line contains one of the three values – increase/decrease/either. The metric is failed (non-Pass classification of High or Low)  if there is a statistical difference found between the canary and baseline metrics in the direction specified by the user"
+        },
+        {
+            "name": "confidenceLevel",
+            "type": "string",
+            "label": "Confidence level",
+            "required": false,
+            "groupName": "classificationInputs",
+            "helpMarkDown": "If unspecified, default value is 0.95"
+        },
+        {
+            "name": "allowedIncrease",
+            "type": "multiLine",
+            "label": "Allowed increase",
+            "required": false,
+            "properties": {
+                "resizable": "true",
+                "rows": "4"
+            },
+            "groupName": "classificationInputs",
+            "helpMarkDown": ""
```

Comment:
Add hel for these inputs? (Or are they self explanatory 🤔)

Reply:
We should have I feel.

Reply:
Tracking it here: #13106